### PR TITLE
Feature/dark theme

### PR DESCRIPTION
**Pull request by nikita-dnestrov**
I dont understand how to set up rerender without reloading the page. For now theme is based on page html tag attribute called data-theme and I am thinking that is a problem. If I'll give data-theme atrribute to, for example, div with id root, the styles mess up and dont work properly. Redux side of this feature seems to work fine and state successfully changes in store. Look for a tip on how to approach this

### CODE REVIEW

File: client/src/components/pages/home/home-page.module.scss [hunk 1]
```diff
@@ -1,9 +1,10 @@
 // will be removed after connecting local fonts
 
 .container {
+	background-color: var(--background-color);
 	width: 916px;
 	font-family: 'Montserrat', Arial, sans-serif;
```
Comment (Paul-Vasylenko) [inline client/src/components/pages/home/home-page.module.scss:6]: would be good if you change it to variable

File: client/src/containers/example/index.tsx [hunk 2]
```diff
@@ -81,4 +89,9 @@ const Example: React.FC = () => {
 	);
 };
 
-export default Example;
+function mapStateToProps(state: any) {
+	const { theme } = state;
+	return { theme };
+}
```
Comment (Paul-Vasylenko) [inline client/src/containers/example/index.tsx:95]: I don't think this is important, but why do you use connect if you can use useSelector? Also, I thought we use connect function only in class components (not in React.FC), but maybe I am wrong.

File: client/src/containers/private-route/index.tsx [hunk 3]
```diff
@@ -14,8 +14,9 @@ interface IPrivateRouteProps extends RouteProps {
 
 const PrivateRoute = (props: IPrivateRouteProps) => {
 	const { component: Component, ...rest } = props;
-	const { user } = useAppSelector((state) => state.auth.userData);
-	const isAuthorized = Boolean(user);
+	// const { user } = useAppSelector((state) => state.auth.userData);
+	// const isAuthorized = Boolean(user);
+	const isAuthorized = true;
```
Comment (Paul-Vasylenko) [inline client/src/containers/private-route/index.tsx:19]: delete this please.

File: client/src/containers/profile/mocks.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@ export const mockProfileBioProps = {
 	followingQuantity: 0,
 	followersQuantity: 0,
 	communityQuantity: 3,
-	rank: 423,
+	rank: 8,
 	score: 222,
```
Comment (Paul-Vasylenko) [inline client/src/containers/profile/mocks.ts:13]: then score should be 0 I suppose :)

**Review by Paul-Vasylenko — CHANGES_REQUESTED** [review]

**Comment by Paul-Vasylenko** [comment]
However you've done nice work :) Small fixes and it's done

File: client/src/containers/private-route/index.tsx (hunk 3 above)
Comment (nikita-dnestrov) [reply client/src/containers/private-route/index.tsx:19]: Ah yes, I always forget it

**Review by Paul-Vasylenko — APPROVED** [review]

**Pull request merged**